client/packages/invoices/src/OutboundShipment/DetailView/AddFromScannerButton.tsx

```diff
@@ -89,6 +87,8 @@ export const AddFromScannerButtonComponent = ({
     [isScanning]
   );
 
+  if (!isEnabled) return null;
+
```

Comment:
This might need testing? For example, is it ok to call stockScan when `isEnabled === false`?